src/main.js

```diff
@@ -84,7 +90,7 @@ function savePoster() {
   var duplicate = checkDuplicate();
   if (!duplicate) {
       savedPosters.push(currentPoster);
-      var newPoster = `${savedPosterLocation.innerHTML} <article class="poster">
+      var newPoster = `${savedPosterLocation.innerHTML} <article class="poster saved-poster" id=${savedPosters.length -1}>
```

Comment:
Standard formatting will have a space between the minus sign and the one: `savedPosters.length - 1`.